src/main/main.ts

```diff
@@ -91,11 +91,11 @@ const installExtensions = async () => {
 };
 
 app.on('will-quit', () => {
-  unmountDrive();
+  unmountDrive().catch(Logger.error);
```

Comment:
can you also call reportError() to report this to Sentry?, this way we can see how many users are getting errors here